src/Helpers/EntityManagerHelper.php

```diff
@@ -28,6 +29,16 @@ public function __construct(ContainerInterface $container)
         $this->container = $container;
     }
 
+    /**
+     * {@inheritdoc}
+     *
+     * @throws \EoneoPay\Externals\ORM\Exceptions\ORMException
+     */
+    public function findAllIds(string $class, array $ids): array
+    {
+        return (new EntityManager($this->getDoctrineEntityManager()))->findByIds($class, $ids);
```

Comment:
As mentioned in slack, DI was avoided on purpose because the EntityManager interface (EoneoPay) may not be bound in the container for this package's usage.
Should it be up to the implementing application to bind that interface? or shall it be done within this search provider

Reply:
or, use `$this->container->make(EntityManagerInterface::class);`